app/src/main/res/values-night/colors.xml

```diff
@@ -8,6 +8,7 @@
     <color name="color_primary">#ef5350</color>
     <color name="color_on_primary">#ffffff</color>
     <color name="color_background">@color/color_background_dark</color>
+    <color name="color_downloaded">#50cf50</color>
```

Comment:
If colors are the same in both light and dark themes we can define them only in values/colors.xml. Also, I think it needs a bit more descriptive name, to indicate that it is there to be used specifically as color code for downloaded torrents (like color_dowloaded_torrents or something. I'm bad with names).

Reply:
I did make them subtly different, but I will renamed them to `color_downloaded_torrents`. :slightly_smiling_face: